monochart/values.yaml

```diff
@@ -174,6 +174,10 @@ deployment:
     lifecycle: {}
     hostPID: false
     terminationGracePeriodSeconds: 30
+    # topologySpreadConstraints:
+    #   - maxSkew: 1
+    #     topologyKey: topology.kubernetes.io/zone
+    #     whenUnsatisfiable: ScheduleAnyway
```

Comment:
Are these safe enough that we'd just want this to always be the default?

Reply:
> Are these safe enough that we'd just want this to always be the default?

It seems like what the scheduler does already now, so I would think it's good, but we should research it more and make sure there aren't any problems. I added comment to the spike ticket: https://spotonteam.atlassian.net/browse/DEV-7358?focusedCommentId=952422